WikilogUtils.php

```diff
@@ -210,7 +210,8 @@ public static function parsedArticle( Title $title, $feed = false ) {
 		$parser->startExternalParse( $title, $parserOpt, Parser::OT_HTML );
 
 		# Parse article.
-		$arttext = $article->getText();
+		$articleContent = $article->getContent();
+		$arttext = $articleContent instanceof TextContent ? $articleContent->getNativeData() : false;
```

Comment:
Is false really a good default value? What do you think about checking, if the content is a TextContent, and if not, returning with an empty string as the text, as it would be pointless to proceed in the codepath with an empty string I think.

return array( $article, '');

Reply:
okay